internal/cmd/fluxtest-harness-influxdb/test.go

```diff
@@ -108,7 +116,36 @@ func (t *testExecutor) Run(pkg *ast.Package) error {
 	}
 
 	// Execute the read pass.
-	return t.executeWithOptions(bucketOpt, orgOpt, t.readOptAST, pkg)
+	err = t.executeWithOptions(bucketOpt, orgOpt, t.readOptAST, pkg)
+	if flags.wait {
+		// TODO(nathanielc): When the executor is given access to the test name,
+		// make the configName a function of the test name.
```

Comment:
Do we need an issue for this, to track the work?

Reply:
I don't think its large enough to warrant an issue. Rather next person the code can update it once test names are a thing.